### PR TITLE
export the CONDA_NPY variable when numpy in engine

### PR DESCRIPTION
**Pull request by PeterDSteinberg**
Refers to this #151 .  When numpy is in the engine in .binstar.yml, take the version and format it for the CONDA_NPY variable, e.g. CONDA_NPY=19 for numpy 1.9.

### CODE REVIEW

**Comment by PeterDSteinberg** [comment]
This is failing just on Windows 2.7 for the same reason #147 PR is failing on Windows 2.7. The failing on Windows 2.7 seems unrelated to either PR.

File: binstar_build_client/worker/utils/data/build_script.bat [hunk 1]
```diff
@@ -221,24 +221,30 @@ goto:eof
...
 
-    echo where conda 
+    echo where conda
     where conda
 
     if "%CONDA_PY%" == "" (
         :: Hack to build with the python set in BINSTAR_ENGINE
-        python -c "import sys; sys.stdout.write('{0}{1}'.format(sys.version_info[0], sys.version_info[1]))" > %TEMP%\CONDA_PY 
-        set /p CONDA_PY=<%TEMP%\CONDA_PY        
+        python -c "import sys; sys.stdout.write('{0}{1}'.format(sys.version_info[0], sys.version_info[1]))" > %TEMP%\CONDA_PY
```
Comment (srossross) [inline binstar_build_client/worker/utils/data/build_script.bat]: Why is this required? do you think it is ok to let `CONDA_PY` be empty? what if numpy is not installed?

File: binstar_build_client/worker/utils/data/build_script.sh [hunk 2]
```diff
@@ -105,9 +105,11 @@ setup_build(){
     if ["$CONDA_PY" == ""]; then
         export CONDA_PY=`python -c 'import sys; sys.stdout.write("{0}{1}".format(sys.version_info[0], sys.version_info[1]))'`
     fi
-
+    if [ "$CONDA_NPY" == "" ];then
+        export CONDA_NPY=$(python -c "import sys;import numpy;sys.stdout.write(''.join(numpy.__version__.split('.')[:2]))" || echo "")
```
Comment (PeterDSteinberg) [inline binstar_build_client/worker/utils/data/build_script.sh]: @srossross if numpy is not installed the python errors out and there is a nonzero return code causing the second part of the statement ```|| echo ""``` to assign CONDA_NPY to "".

**Comment by srossross** [comment]
:+1:

**Comment by srossross** [comment]
@PeterDSteinberg can you co-ordiate with @electronwill to get this new functioality added to the docs?

**Comment by PeterDSteinberg** [comment]
Yes will do.  (Offline for most of the afternoon now)
